src/org/skypro/skyshop/exceptions/BestResultNotFound.java

```diff
@@ -0,0 +1,9 @@
+package org.skypro.skyshop.exceptions;
+
+import java.io.IOException;
+
+public class BestResultNotFound extends IOException {
```

Comment:
IOException это Input Output Exception - ошибка ввода/вывода. Ввода и вывода у тебя нет, нужно наследоваться от класса Exception

```suggestion
public class BestResultNotFound extends Exception {
```